### PR TITLE
 separate action on clicking vod and live video player

### PR DESCRIPTION
**Pull request by iw2d-rn**
issue : https://github.com/SevenTV/Extension/issues/762

### CODE REVIEW

File: src/site/twitch.tv/modules/player/PlayerController.vue [hunk 1]
```diff
@@ -68,12 +68,26 @@ watchEffect(() => {
...
 
-		if (actionOnClick.value === 1) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", togglePause);
-		} else if (actionOnClick.value === 2) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", toggleMute);
-		} else {
-			unsetNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click");
+		const isLiveOrVOD =
+			props.inst.component.props.children.props.children.props.children.props.children.props.children.props
+				.children.props.children.props.children.props.children.props.children.props.content.type;
```
Comment (iw2d-rn) [inline src/site/twitch.tv/modules/player/PlayerController.vue:73]: how can i write it better. i could not think of any thing :sweat_smile:

File: src/site/twitch.tv/modules/player/PlayerController.vue [hunk 2]
```diff
@@ -68,12 +68,31 @@ watchEffect(() => {
 		);
 		if (!playerOverlay.value) return;
 
-		if (actionOnClick.value === 1) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", togglePause);
-		} else if (actionOnClick.value === 2) {
-			defineNamedEventHandler(playerOverlay.value, "PlayerActionOnClick", "click", toggleMute);
+		// using xpath to get the `LIVE` indicator on the player to determine if channel is live or not
+		const isLiveOrVOD = document.evaluate(
+			"//div[contains(@class, 'persistent-player')]//div[contains(@class, 'tw-channel-status-text-indicator')]/p",
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/player/PlayerController.vue:73]: It's unclear why use an extremely old web technology when better alternatives, such as `document.querySelector()`, exist.

**Review by AnatoleAM — CHANGES_REQUESTED** [review]
Sorry, you're gonna have to find a better way to do this. This is not

**Comment by TroyKomodo** [comment]
Features against this repo wont be implemented, we are going to rewrite the extension on this [repo](https://github.com/SevenTV/SevenTV).

**Pull request closed without merging**